tasks/gen/templates/resource/actions/{{folder_name}}/delete.cr.ecr

```diff
@@ -1,7 +1,9 @@
 class <%= pluralized_name %>::Delete < BrowserAction
   <%= route("Delete") %> do
-    <%= query_class %>.find(<%= resource_id_method_name %>).delete
-    flash.success = "Deleted the record"
-    redirect Index
+    <%= underscored_resource %>  = <%= query_class %>.find(<%= resource_id_method_name %>)
+    <%= resource %>::DeleteOperation.destroy(<%= underscored_resource %>) do |operation, deleted|
```

Comment:
```suggestion
    <%= resource %>::DeleteOperation.destroy(<%= underscored_resource %>) do |_operation, _deleted|
```

Since these two variables aren't used in the block, adding the `_` in front will keep [Ameba](https://github.com/crystal-ameba/ameba) from throwing any errors.